generators/FairYPtGenerator.cxx

```diff
@@ -46,8 +46,8 @@ void FairYPtGenerator::SetYPt(const TH2D &yPt)
 
 Bool_t FairYPtGenerator::Init()
 {
-    if (FairBaseMCGenerator::Init() == kFALSE)
-        return kFALSE;
+    if (!FairBaseMCGenerator::Init())
+        return fFALSE;
```

Comment:
Typo…

I wonder, why this compiles on some of our machines.

```suggestion
        return false;
```

Reply:
> I wonder, why this compiles on some of our machines.

I suspect overlapping success signals from the previous CI builds.

Reply:
Ya, that's it, https://alfa-ci.gsi.de/blue/organizations/jenkins/FairRootGroup%2FFairRoot/detail/PR-1267/3/pipeline/ here you see that all really failed.